components/scream/cmake/machine-files/generic-desktop-debug.cmake

```diff
@@ -1,5 +0,0 @@
-set(KOKKOS_ENABLE_DEBUG TRUE CACHE BOOL "")
-set(KOKKOS_ENABLE_AGGRESSIVE_VECTORIZATION FALSE CACHE BOOL "")
-set(KOKKOS_ENABLE_OPENMP TRUE CACHE BOOL "")
```

Comment:
Why was this deleted?

Reply:
It got deleted during a rebase. Thanks for catching that. I have reverted this change.